### DISCUSSION TITLE
wifi support

### DISCUSSION THREAD

**Discussion by Sangra69** [feelfreelinux/octo4a#74] [Ideas]
Would love to be able to connect my wifi on my printer to the phone. That way octoprint will have an easier access to the 3d printer.

**Comment by feelfreelinux** [comment]
What printer are you using with octo4a?

**Reply by ddv5725910** [reply]
First want to say thank you for the awesome work.

I have a "**Two trees sapphire plus**" printer running "**BIGTREETECH SKR Pro v1.2**" Motherboard,  with a "**ESP-01S Serial Wireless WiFi Module**"

I'll be glad to help do some testing if needed.

Cheers
